.github/workflows/release-notes.yml

```diff
@@ -0,0 +1,36 @@
+# This code is a Qiskit project.
```

Comment:
File should be something like `external-link-checker.yml`

Reply:
Done!